services/api.ts

```diff
@@ -0,0 +1,17 @@
+import { AxiosError } from 'axios';
+
+import apiClient from '@/libs/axios';
+
+export async function postSignUpInfo(body: object) {
```

Comment:
이정도로만 함수 이름은 표현해줘도 괜찮을 것 같아요!
```suggestion
export async function postSignUp(body: object) {
```

Reply:
object 타입은 사용하는걸 지양하는게 좋구요! (해당 함수를 사용하는 쪽에서 `{}`와 같이 빈 객체를 넘겨도 에러가 발생하지 않아요!)
아래와 같이 명시적으로 타입을 잡아주고 사용해주는게 좋아요!
```suggestion
interface PostSignUpInfoBody {
  email: string;
  password: string;
  confirmPassword: string;
}

export async function postSignUpInfo(body: PostSignUpInfoBody) {
```